sechub-doc/src/docs/asciidoc/sechub-getting-started.adoc

```diff
@@ -185,6 +186,29 @@ Setup of GoSec complete:
 
 Now you are ready to do scans!
 
+==== Get SecHub client 
+The SecHub client is needed to initiate scans. In later sections of this tutorial, the client is used to do an example scan. In case you don't have it, you cannot continue with the tutorial. The command below, will download the latest version and put it in your `/usr/local/bin` folder.
+
+[source, bash]
+--
+# Get latest version
+CLIENT_VERSION=`curl -s https://mercedes-benz.github.io/sechub/latest/client-download.html | grep https://github.com/mercedes-benz/sechub/ | awk -F '-' '{print $NF}' | sed 's/.zip">//'`
+
+# Download the zipped binary
+curl -L -o sechub-cli.zip https://github.com/mercedes-benz/sechub/releases/download/v$CLIENT_VERSION-client/sechub-cli-$CLIENT_VERSION.zip
+
+# Verify the binary
+curl -L -o sechub-cli.zip.sha256 https://github.com/mercedes-benz/sechub/releases/download/v$CLIENT_VERSION-client/sechub-cli-$CLIENT_VERSION.zip.sha256
+sha256sum --check sechub-cli.zip.sha256
+
+# Extract 
+unzip sechub-cli.zip
+sudo cp platform/linux-amd64/sechub /usr/local/bin
```

Comment:
This is only the right command if you are using linux-amd64 (Intel or AMD 64-bit processors). Please add at least a command in the bash script that this might need to be changed for Intel 32-bit or ARM based CPUs.